### PR TITLE
ER-1119: Video bundle paragraphs type

### PR DESCRIPTION
**Pull request by rimi-itk**
https://jira.itkdev.dk/browse/ER-1119

Adds the Video bundle paragraphs type.

### CODE REVIEW

File: ereol_inspiration/ereol_inspiration.features.field_instance.inc [hunk 1]
```diff
@@ -203,10 +203,10 @@ function ereol_inspiration_field_default_field_instances() {
         'newsletter_signup' => 'newsletter_signup',
         'picked_article_carousel' => 'picked_article_carousel',
         'recommended_material' => -1,
-        'newsletter_signup' => 'newsletter_signup',
```
Comment (cableman) [inline ereol_inspiration/ereol_inspiration.features.field_instance.inc]: Why the newslette signup removal?

File: ereol_video/ereol_video.features.inc [hunk 2]
```diff
@@ -40,3 +40,17 @@ function ereol_video_node_info() {
...
   return $items;
 }
+
+/**
+ * Implements hook_paragraphs_info().
+ */
+function ereol_video_paragraphs_info() {
+  $items = array(
+    'video_bundle' => array(
+      'name' => 'Videobundt',
```
Comment (cableman) [inline ereol_video/ereol_video.features.inc]: Danish "Videobundt" should be wrapped in t() function.

File: ereol_video/ereol_video.file_default_displays.inc [hunk 3]
```diff
@@ -64,9 +66,45 @@ function ereol_video_file_default_displays() {
...
+  );
   $export['video__video_show__file_field_media_large_icon'] = $file_display;
 
+  $file_display = new stdClass();
+  $file_display->api_version = 1;
+  $file_display->name = 'video__video_show__media_videotool_image';
+  $file_display->weight = 0;
+  $file_display->status = FALSE;
+  $file_display->settings = array(
+    'image_style' => '',
```
Comment (cableman) [inline ereol_video/ereol_video.file_default_displays.inc:80]: No image style?

File: ereol_video/ereol_video.info [hunk 4]
```diff
@@ -3,9 +3,12 @@ description = Video content type for eReolen
 core = 7.x
 package = eReolen
 version = 7.x-1.0
-dependencies[] = node
-dependencies[] = reol_base
```
Comment (cableman) [inline ereol_video/ereol_video.info:7]: Removed dependencies?

**Review by cableman — CHANGES_REQUESTED** [review]

File: ereol_video/ereol_video.file_default_displays.inc (hunk 3 above)
Comment (rimi-itk) [reply ereol_video/ereol_video.file_default_displays.inc:80]: No. It's a video, not an image. The other video formats (youtube and vimeo) don't have image styles either.

File: ereol_video/ereol_video.features.inc (hunk 2 above)
Comment (rimi-itk) [reply ereol_video/ereol_video.features.inc]: Nice catch. Fixed.

File: ereol_video/ereol_video.info (hunk 4 above)
Comment (rimi-itk) [reply ereol_video/ereol_video.info:7]: Features suck. Fixed.

File: ereol_inspiration/ereol_inspiration.features.field_instance.inc (hunk 1 above)
Comment (rimi-itk) [reply ereol_inspiration/ereol_inspiration.features.field_instance.inc]: Features suck. Fixed.

**Review by cableman — APPROVED** [review]

**Pull request merged**